AddTransaction.cs

```diff
@@ -17,7 +17,14 @@ public AddTransaction(UserData userData, ITransactionService transactionService)
 
         private void AddTransactionButton_Click(object sender, EventArgs e)
         {
-            transactionService.AddNewTransaction(ComboBoxTransactionType.Text, TextBoxTitleName.Text, TextBoxTransactionAmount.Text, ComboBoxTransactionCategory.Text);               
+            if(ComboBoxTransactionCategory.Text == "")
+            {
+                transactionService.AddNewTransaction(transactionType: ComboBoxTransactionType.Text, transactionName: TextBoxTitleName.Text, transactionAmount: TextBoxTransactionAmount.Text);
+            }
+            else
+            {
+                transactionService.AddNewTransaction(category: ComboBoxTransactionCategory.Text, transactionType: ComboBoxTransactionType.Text, transactionName: TextBoxTitleName.Text, transactionAmount: TextBoxTransactionAmount.Text);
+            }
             Close();
```

Comment:
For future, look up Decorator or Builder design patterns, cause we'll probably need more flexibility with this, cause these 6 line if statements are ugly